assets/js/googlesitekit/constants.js

```diff
@@ -19,3 +19,10 @@
 export const VIEW_CONTEXT_DASHBOARD = 'dashboard';
 export const VIEW_CONTEXT_PAGE_DASHBOARD = 'pageDashboard';
 export const VIEW_CONTEXT_POSTS_LIST = 'postsList';
+export const VIEW_CONTEXT_USER_INPUT = 'userInput';
+export const VIEW_CONTEXT_ACTIVATION = 'activation';
+export const VIEW_CONTEXT_DASHBOARD_SPLASH = 'splash';
+export const VIEW_CONTEXT_ADMIN_BAR = 'adminBar';
+export const VIEW_CONTEXT_SETTINGS = 'settings';
+export const VIEW_CONTEXT_MODULE = 'module';
+export const VIEW_CONTEXT_WP_DASHBOARD = 'WPDashboard';
```

Comment:
While this is what's in the IB too, let's go with `wpDashboard`, since `WPDashboard` is pascal-case rather than camel-case which is what we use for all the other ones. Regarding our "acronyms must be capitalized" rule, that only applies for acronyms that are not the first segment of a camel-case name. cc @eugene-manuilov